node-data/src/message.rs

```diff
@@ -73,8 +73,8 @@ impl Serializable for Message {
 
         match &self.payload {
             Payload::NewBlock(p) => p.write(w),
```

Comment:
Shound `NewBlock` be renamed?

Reply:
Good point! Payload::NewBlock was renamed to Payload::Candidate